project-based/docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+export HOME=/fake-home
+
+rm -rf /work/node_modules
+cp -rf /node_modules /work
```

Comment:
maybe better to use fake home as a place where node modules will live?

Reply:
Moved to /fake-root